spyder/plugins/editor.py

```diff
@@ -1238,7 +1238,7 @@ def register_editorstack(self, editorstack):
             self.set_last_focus_editorstack(self, editorstack)
             editorstack.set_closable( len(self.editorstacks) > 1 )
             if self.outlineexplorer is not None:
-                editorstack.set_outlineexplorer(self.outlineexplorer)
+                editorstack.set_outlineexplorer(self.outlineexplorer.explorer)

```

Comment:
Fix Issue 1

Reply:
This was my bad in a refactoring I did some months. Thanks for catching it.